uPortal-api/uPortal-api-platform-impl/src/main/java/org/apereo/portal/api/groups/EntityService.java

```diff
@@ -48,76 +49,112 @@ public static EntityService instance() {
     }
 
     public Set<Entity> search(String entityType, String searchTerm) {
+        if (!StringUtils.isEmpty(entityType) && !StringUtils.isEmpty(searchTerm)) {
```

Comment:
Consider `StringUtils.isNotEmpty(x)` rather than `!StringUtils.isEmpty(x)`. Syntactic sugar for the win.
Consider `isBlank()` rather than `isEmpty()` so as to do the right thing with Strings consisting purely of whitespace.